tests/set-annotations/additional-annotations/.expected/results.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: kpt.dev/v1
+kind: FunctionResultList
+metadata:
+  name: fnresults
+exitCode: 0
+items:
+  - image: gcr.io/kpt-fn/set-annotations:unstable
+    exitCode: 0
+    results:
+      - message: set annotation value to "orange"
+        field:
+          path: data.selector.annotations.color
+        file:
+          path: local-config.yaml
```

Comment:
`file` also has a field called `index`.
We should add it for completeness.

Reply:
I don't object to adding it here, although from my testing it seems to have no impact on the user facing output

Reply:
done